maui/src/main/java/org/testmonkeys/maui/core/elements/actions/ClearInputAction.java

```diff
@@ -0,0 +1,16 @@
+package org.testmonkeys.maui.core.elements.actions;
+
+import org.openqa.selenium.WebElement;
+import org.testmonkeys.maui.pageobjects.elements.AbstractComponent;
+
+public class ClearInputAction extends AbstractVoidAction {
+
+    public ClearInputAction(AbstractComponent component) {
+        super(component);
+    }
+
```

Comment:
Please override describe method as well, that's used for trace logging

Reply:
done